app/components/quiz/QuizQuestion.js

```diff
@@ -1,29 +1,74 @@
 import React from 'react';
 import QuizAnswer from 'components/quiz/QuizAnswer.js';
 import QuizContext from 'components/quiz/QuizContext.js';
+import RandomColor from 'utils/palette';
 
 export default React.createClass({
   displayName: 'QuizQuestion',
 
   propTypes: {
     question: React.PropTypes.object.isRequired,
-    onUpdate: React.PropTypes.func.isRequired
+    onUpdate: React.PropTypes.func.isRequired,
```

Comment:
nit: this causes a warning

I actually like trailing `,` because in new PRs extending a list results in one line change instead of two line changes, but our ESLint config disallows this.